thorlcr/activities/thdiskbaseslave.cpp

```diff
@@ -119,7 +119,7 @@ void CDiskPartHandlerBase::open()
     {
         StringBuffer locations;
         IException *e = MakeActivityException(&activity, TE_FileNotFound, "No physical file part for logical file %s, found at given locations: %s (Error = %d)", activity.logicalFilename.get(), getFilePartLocations(*partDesc, locations).str(), GetLastError());
-        EXCLOG(e, NULL);
+        ERRLOG(e, "CDiskPartHandlerBase::open()");
```

Comment:
Looks like you made this comment on the 1st commit, it was changed from ERRLOG to IERRLOG in the 3rd commit.
I thought programmer audience vs user is appropriate, since the exception is being fired and propagated back to the user.